tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -165,5 +165,25 @@ string GetLoadedTensorRTVersion() {
   return absl::StrCat(major, ".", minor, ".", patch);
 }
 
+int GetNumberOfEngineInputs(
```

Comment:
Why this routine is outside #if GOOGLE_CUDA && GOOGLE_TENSORRT?
I am getting this compilation error
tensorflow/compiler/tf2tensorrt/convert/utils.cc:168:35: error: use of undeclared identifier 'nvinfer1'
int GetNumberOfEngineInputs(const nvinfer1::ICudaEngine* engine) {
                                  ^
tensorflow/compiler/tf2tensorrt/convert/utils.cc:179:5: error: function-like macro 'IS_TRT_VERSION_GE' is not defined
#if IS_TRT_VERSION_GE(6, 0, 0, 0)
    ^

when building target without --config=cuda tensorflow/compiler/tf2tensorrt:trt_shape_optimization_profiles_test

Reply:
That is a mistake, I have fixed it.